pkg/cmd/grafana-cli/commands/conflict_user_command.go

```diff
@@ -33,7 +51,12 @@ func getSqlStore(context *cli.Context) (*sqlstore.SQLStore, error) {
 
 func runListConflictUsers() func(context *cli.Context) error {
 	return func(context *cli.Context) error {
-		s, err := getSqlStore(context)
+		cmd := &utils.ContextCommandLine{Context: context}
+		cfg, err := initConflictCfg(cmd)
+		if err != nil {
```

Comment:
*nit

This and the following 3 conditional statements are identical in `runListConflictUsers`, `runGenerateConflictUsersFile`, `runGenerateConflictUsersFile` and `runIngestConflictUsersFile`.

Would it make sense to abstract the conditional statements into a shared function or move them to `initConflictCfg`?